hack/redhat.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Cockroach Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -euo pipefail
+
+# include bazel binaries in the path
+PATH="bazel-bin/hack/bin:${PATH}"
+
+# Global preflight settings
+export PFLT_DOCKERCONFIG="${HOME}/.docker/config.json"
+export PFLT_LOGLEVEL=debug
+
+OPERATOR="cockroachdb/cockroach-operator"
+REGISTRY="gcr.io/${GCP_PROJECT}"
+VERSION="$(cat version.txt)"
+IMAGE="${OPERATOR}:v${VERSION}"
+BUNDLE_IMAGE="${OPERATOR}-bundle:v${VERSION}"
+BUNDLE_INDEX="${OPERATOR}-index:v${VERSION}"
+RHMP_BUNDLE_IMAGE="${OPERATOR}-bundle-rhmp:v${VERSION}"
+RHMP_BUNDLE_INDEX="${OPERATOR}-index-rhmp:v${VERSION}"
+
+main() {
+  # Switch to the build directory. The bundle directory is not part of source
+  # controller and therefore isn't a bazel target. This means when we run this
+  # script, there's no way to reference the Dockerfile created by the call to
+  # make release/generate-bundle (prerequisite of make test/preflight). By cd'ing
+  # into the build directory, we'll have access to _all_ the files.
+  if [[ -n "${BUILD_WORKSPACE_DIRECTORY}" ]]; then
+    cd "${BUILD_WORKSPACE_DIRECTORY}"
+  fi
+
+  case "${1:-}" in
+    operator)
+			publish_operator_image
+      preflight_operator;;
+    bundle)
```

Comment:
/nit again, formatting. Typically 2 space indent for bash files. Bonus points if the whole file uses the same 😄

Reply:
Done

Reply:
Done